main.css

```diff
@@ -0,0 +1,469 @@
+  text-align: center;
+}
+
+.tool-column svg {
+  width: 1.5em;
+  height: 1.5em;
+  fill: gray;
+  padding: 0.5em;
+  border-radius: 50%;
+}
+
+.tool-column svg:hover {
+  background-color: lightgray;
+  fill: white;
+}
+
+
+/* Right Pane - Aside with suggestions and recommendations */
+
+.additional-pane {
+  grid-area: additional-pane;
+  border-left: 1px lightgray solid;
+  height: 100vh;
+  position: relative;
+  top: 0;
+  left: 0;
+}
+
+.search {
+  margin-top: 1em;
+}
+
+.searchbar {
+  width: 80%;
+  height: 2em;
+  padding: 1em;
+  color: gray;
+  background-color: #eee;
+  border: 1px lightgray solid;
+  border-radius: 2em;
+  margin-bottom: 1em;
+}
+
+.recommendations {
+  padding: 1em 0 1em 0;
+  border-radius: 1em;
+  background-color: #eee;
+}
+
+.recommendations h2 {
+  padding-left: 0.5em;
+}
+
+.suggested-article {
+  display: grid;
+  grid-template:
+    "category-meta-data more"
+    "news-headline suggested-article-img";
+  gap: 0.5em;
+  padding: 1em;
+}
+
+.suggested-article:hover {
+  background-color: #ddd;
+}
+
+.category-meta-data {
+  grid-area: category-meta-data;
+  color: gray;
+}
+
+.more {
+  grid-area: more;
+  text-align: center;
+  font-weight: bold;
+}
+
+.news-headline {
+  grid-area: news-headline;
+}
+
+.suggested-article-img {
+  grid-area: suggested-article-img;
+  margin-top: 1em;
+}
+
+.suggested-article-img img {
+  border-radius: 0.5em;
+  width: 80px;
+}
+
+.direct-messages {
+  /* min-width: 120px; */
+  padding: 1em;
+  border: 1px lightgray solid;
+  background-color: white;
+  border-radius: 1em 1em 0 0;
+  box-shadow: 0px 0px 6px gray;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  gap: 1em;
+  position: absolute;
+  bottom: 2em;
+  left: 1em;
+}
+
+.messages-label {
+  padding-left: 1em;
+  font-weight: bold;
+}
+
+.messages-tools {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin-top: 0.5em;
+}
+
+.messages-tool {
+  text-align: center;
+}
+
+.message-tool svg {
+  width: 1.5em;
+  height: 1.5em;
+  fill: gray;
+  padding: 0.5em;
+  border-radius: 50%;
+}
+
+.message-tool svg:hover {
+  background-color: #ccc;
+}
+
+/* Fixed compliance floater */
+
+#gdpr-compliance {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  background-color: black;
+  color: white;
+  padding: 1em;
+  gap: 1em;
+  z-index: 2;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+#gdpr-compliance a {
+  color: lightblue;
+}
+
+#gdpr-compliance a:hover {
+  color: white;
+}
+
+.gdpr-buttons {
+  padding: 1em;
+}
+.gdpr-button {
+  width: 100%;
+  margin: 0.5em;
+  padding: 1em;
+  color: black;
+  background-color: lightgray;
+  border-radius: 1em;
+  font-weight: bold;
+  border: 0;
+}
+
+.gdpr-button:hover {
+  background-color: gray;
+}
+
+.instructions-for-teacher {
+  color: red;
+}
+
+/* Responsive settings for smaller screen sizes */
+
+@media only screen and (max-width: 1400px) {
+  #container {
+    width: 100%;
+  }
+}
+
+@media only screen and (max-width: 1000px) {
```

Comment:
Maybe you could have had one more media query with only the left pane and main area for smaller screens.

Reply:
Thank you. That is a great suggestion. I've now implemented and pushed it.